src/test/java/org/broadinstitute/hellbender/engine/GenomicsDBIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+package org.broadinstitute.hellbender.engine;
+
+import genomicsdb.GenomicsDBUtils;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.walkers.variantutils.SelectVariants;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.runtime.ProcessController;
+import org.broadinstitute.hellbender.utils.runtime.ProcessOutput;
+import org.broadinstitute.hellbender.utils.runtime.ProcessSettings;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.function.BiConsumer;
+
+public class GenomicsDBIntegrationTest extends CommandLineProgramTest {
+
+    public static final String TEST_DATA_PATH =  "src/test/resources/org/broadinstitute/hellbender/engine/GenomicsDBIntegration/";
+    private static final String TINY_GVCF = TEST_DATA_PATH + "tiny_jsons/tiny.gvcf";
+    private static final String WORKSPACE = TEST_DATA_PATH + "workspace";
+    private static final String GENOMICS_DB_JSONS = "gendb://" + TEST_DATA_PATH + "tiny_jsons";
+
+    public static final String GENOMICSDB_BIN_PATH_PROPERTY_NAME = "GATK_GENOMICSDB_BIN";
+
+
+    @FunctionalInterface
+    public interface GenomicsDBTestAction {
+        void run() throws IOException;
+    }
+
+    public synchronized static void runOnGenomicsDBArray(GenomicsDBTestAction action) throws IOException {
+        final File workspace = new File(WORKSPACE);
```

Comment:
can't use a temp location because i need to hardcode the paths in the json files.  It would be possible to edit them, but it doesn't seem worth the complexity.